scripts/start-servers.php

```diff
@@ -47,30 +47,39 @@ function make_ctx($preset, $method = "POST") {
     return $ctx;
 }
 
-function failed($result) {
-    echo "\n\n";
-    echo join("\n", $result);
-    printf("Last operation took: %.2f secs\n", lap());
-    exit();
-}
-
 function mo_http_request($uri, $context) {
     global $http_response_header;
 
     $result = file_get_contents($uri, false, $context);
 
     if ($result === false) {
-        failed($http_response_header);
+        printf("HTTP request to %s failed:\n", $uri);
+        var_dump($http_response_header);
+        printf("Last operation took: %.2f secs\n", lap());
+        exit(1);
     }
 
     return $result;
 }
 
+function json_decode_or_fail(/* json_decode() args */)
```

Comment:
No variadics for here and the call to ``json_decode`` below? http://php.net/manual/en/functions.arguments.php#functions.variable-arg-list

Reply:
Not until we drop PHP 5.5 compatibility.

Reply:
"start-servers.php" doesn't have to be PHP 5.5 compatible though...